lib/ccf/functional_preprocessing/one_subject_prereq_checker.py

```diff
@@ -6,6 +6,10 @@
 
 # import of local modules
```

Comment:
Changes in this file are for the other feature: invoking functions from a local system instead of from the CHPC